### PR TITLE
Adding StealthChain

### PR DESCRIPTION
**Pull request by degenstealthdev**
We'd Be Pleased to get listed on L2beat as a Up and coming Launchpad!

### CODE REVIEW

File: packages/config/src/layer2s/stealthchain.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    name: 'StealthChain',
+    slug: 'stealthchain',
+    description:
+      'Degen Stealth Launchpad, Cross Chain Token Bridge & LP Pool',
+    purpose: 'Degen,Launchpad',
+    category: 'OP Stack',
+    links: {
+      websites: ['https://steathchain.org'],
+      apps: ['https://bridge.test.steathchain.org'],
+      documentation: ['https://docs.steathchain.org'],
```
Comment (mateuszradomski) [inline packages/config/src/layer2s/stealthchain.ts:20]: These links are dead

- `steathchain.org` is up available to be bought
- `stealthchain.org` was bought today

So I presume it's a typo, but make sure that the website and docs work after DNS propagates

**Review by mateuszradomski — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/stealthchain.ts (hunk 1 above)
Comment (degenstealthdev) [reply packages/config/src/layer2s/stealthchain.ts]: Yes Typo Links Updated.

Thank you

**Comment by degenstealthdev** [comment]
Any Update on this PR?

**Comment by lucadonnoh** [comment]
Moved here since i cannot push https://github.com/l2beat/l2beat/pull/1890

**Pull request closed without merging**